clientapi/ratelimit/rt_failed_login_test.go

```diff
@@ -0,0 +1,40 @@
+package ratelimit
+
+import (
+	"testing"
+	"time"
+
+	"github.com/matryer/is"
+)
+
+func TestRtFailedLogin(t *testing.T) {
+	is := is.New(t)
+	dut := NewRtFailedLogin(&RtFailedLoginConfig{
```

Comment:
Why did you choose the `dut` naming for this `RtFailedLogin` variable? 🤔

Reply:
I realized it does makes sense in this context: https://en.wikipedia.org/wiki/Device_under_test, I will change it to `rtfl`